go/memcached.go

```diff
@@ -27,22 +28,46 @@ func newMemcache(c *config) task {
 }
 
 func (m *memcache) init() {
-	if len(m.config.dataBytes) != 0 {
-		m.client.Set(&gomemcache.Item{Key: m.config.key, Value: m.config.dataBytes})
+	if m.config.dataRange[0] != 0 {
```

Comment:
with the slice option I mentioned above this could be simplified to:
```
if m.config.dataRange != nil {
  if m.config.keyrange !=0 {
    for 0..keyrange{
      
    } 
  } else {
   m.client.set(key, config.databytes)
  }
}
```